lib/web/ui/usercontext.go

```diff
@@ -197,7 +197,11 @@ func NewUserContext(user types.User, userRoles services.RoleSet, features proto.
 	requestAccess := newAccess(userRoles, ctx, types.KindAccessRequest)
 	desktopAccess := newAccess(userRoles, ctx, types.KindWindowsDesktop)
 	cnDiagnosticAccess := newAccess(userRoles, ctx, types.KindConnectionDiagnostic)
-	assistAccess := newAccess(userRoles, ctx, types.KindAssistant)
+
+	var assistAccess access
+	if features.Assist {
```

Comment:
In the UI, the check is basically: `hasAssistAccess && assistEnabledInConfig`. This change effectively ANDs `hasAssistAccess = hasAssistAccess && features.Assist`.

Reply:
Did you check the UI? From what I remember, webUI "assumes" that this field is always there.

Reply:
Yes, UI works fine. `access` is a value type, so it will just be all `false` by default.